sphinxcontrib/autorunrecord.py

```diff
@@ -165,11 +165,13 @@ def get_code(self, encode=False):
         return code
 
 
-    def write_cast(self, capture_file_cast):
+    def write_cast(self, capture_file_cast, castcount):
         """Write a cast from tagged code examples"""
         import shlex
         notes = self.options.get('notes', None)
         code = self.get_code(encode=False)
+        counter = '### Code snippet {}\n'.format(castcount)
+        code = counter + code
```

Comment:
I would merge the two lines into a single format() call

Reply:
oh, yes, that makes sense.